### ISSUE TITLE
fixed length array

### ISSUE THREAD

**Issue by visko-sc** [feliixx/mgodatagen#75]
Currently, an array takes only `generator` arrayContent. There are two problems with it:
- it is not possible to generate mixed-value list: ['one', 2]
- it is not possible to define an array with constant values: ['one', 'two'] - although workaround could be possible with Enum support, it would be better to have support for constant values in an array:

```
"fieldName": {
    "type": "array",
    "arrayContent": [
        {
             "type": "constant",
             "constVal": "one"
         },
        {
             "type": "constant",
             "constVal": 2
         }
    ]
}
```

**Comment by feliixx** [comment]
Yes, this is currently not possible :/ Mixed-value list never felt right to me, so I didn't implemented it, but if you want feel free to send a PR !
